src/hooks/useSettingsUtilities.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  useDeskproAppClient,
+  useDeskproLatestAppContext,
+} from "@deskpro/app-sdk";
+import { IBookmark } from "../types/bookmarks";
+
+export const useSettingsUtilities = () => {
+  const { client } = useDeskproAppClient();
+  const { context } = useDeskproLatestAppContext();
+
+  if (!client || !context) return null;
+
+  const getBookmarks = (): IBookmark[] => {
+    return JSON.parse(
+      context.settings.bookmarks ||
+        '[{"Id": "f238cf6d-eb4e-4873-99b9-fb5c2443820c", "Name": "Root", "URL": "", "Description": "", "ParentFolder": null, "isFolder": true}]'
```

Comment:
Is it correct that here is a hardcoded `Id`?

Reply:
Yea but it could have lit any value, doesnt make a difference